virtool/settings/oas.py

```diff
@@ -0,0 +1,54 @@
+from pydantic import BaseModel, Field, constr
+from virtool_core.models.settings import Settings
+
+
+class GetSettingsResponse(Settings):
+    class Config:
+        schema_extra = {
+            "example": {
+                "default_source_types": ["isolate", "strain"],
+                "enable_api": True,
+                "enable_sentry": True,
+                "hmm_slug": "virtool/virtool-hmm",
+                "minimum_passsword_length": 8,
+                "sample_all_read": False,
+                "sample_all_write": True,
+                "sample_group": "force_choice",
+                "sample_group_read": True,
+                "sample_group_write": True,
+                "sample_unique_names": False,
+            }
+        }
+
+
+class EditSettingsSchema(BaseModel):
+    sample_group: str = Field(default="none")
+    sample_group_read: bool = Field(default=True)
+    sample_group_write: bool = Field(default=False)
+    sample_all_read: bool = Field(default=True)
+    sample_all_write: bool = Field(default=False)
+    sample_unique_names: bool = Field(default=True)
+    hmm_slug: constr(strip_whitespace=True) = Field(default="virtool/virtool-hmm")
+    enable_api: bool = Field(default=False)
+    enable_sentry: bool = Field(default=True)
+    minimum_password_length: int = Field(default=8)
+    default_source_types: list = Field(default=["isolate", "strain"])
+
+
+class UpdateSettingsResponse(Settings):
```

Comment:
Please make naming consistent. Change everything for updating settings to match this class. For example, `EditSettingsSchema` should be `UpdateSettingsSchema`.

Fix any other places the word 'edit' is used instead of 'update'.